### ISSUE TITLE
This plugin does not work with iOS 13

### ISSUE THREAD

**Issue by sm1th** [rmcfarlane82/react-native-msal-plugin#37]
I'm just opening this issue to make it easy for other people to find.

This plugin will not work on iOS 13 because it is using an old version of the obj-c MSAL library which does not work on iOS 13.

**Comment by vukmandavid** [comment]
Are you sure about this? I tested and it's working on iOS 13 devices.

**Comment by sm1th** [comment]
I... Was pretty sure. Are you sure? Lol.

**Comment by vukmandavid** [comment]
Yes, also a quite big userbase is currently using the app via test flight and no one complained. 

But I just realized I'm still using XCode 10.3. I guess the new version doesn't let you build the app. Sorry for the confusion.
